packages/client-core/src/components/MediaIconsBox/index.tsx

```diff
@@ -204,7 +205,11 @@ export const MediaIconsBox = () => {
           )}
         </>
       ) : null}
-      {screenshareEnabled && mediaNetworkReady && mediaNetworkState?.ready.value ? (
+      {!isMobile &&
+      navigator.mediaDevices.getDisplayMedia != null &&
```

Comment:
can we do `!navigator.mediaDevices.getDisplayMedia` or `typeof navigator.mediaDevices.getDisplayMedia === 'undefined'` instead?